Some edits on key management and  some questions (#26)
uptane/deployment-considerations: key_management.md

In the second option, the OEM uses offline keys to sign Timestamp and Snapshot metadata, which reduces the risk of attackers immediately publishing malicious images. Here again, though, there are trade-offs, in this case related to the metadata expiration dates. If the Timestamp and Snapshot metadata expire relatively quickly, then it may be cumbersome to use offline keys to renew their signatures. However, if a longer expiration time is used, then a man-in-the-middle attacker has more time with which to execute freeze attacks, hence defeating the purpose of the Timestamp role. 

> In most use cases, the online option may be best, but if you want stronger security guarantees, consider using  online for the Timestamp and Snapshot roles.

Reviewer: ```suggestion
In most use cases, the online option may be best, but if stronger security guarantees are desired, consider using  the offline option instead for the Timestamp and Snapshot roles.
```
